package.json

```
@@ -92,7 +93,8 @@
     "typescript": "^5.6.3"
   },
   "resolutions": {
-    "set-value": "^4.0.1"
+    "set-value": "^4.0.1",
+    "cross-spawn": "^7.0.5"
```

```suggestion
    "set-value": "^4.0.1"
```
can you remove this and run `yarn install` again to see if the dependency is updated as expected?